modules/phase_field/include/postprocessors/AverageGrainVolume.h

```diff
@@ -0,0 +1,53 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef AVERAGEGRAINVOLUME_H
+#define AVERAGEGRAINVOLUME_H
+
+#include "GeneralPostprocessor.h"
+#include "Coupleable.h"
+#include "MooseVariableDependencyInterface.h"
+
+// Forward Declarations
+class FeatureFloodCount;
+class AverageGrainVolume;
+class MooseMesh;
+template <>
+InputParameters validParams<AverageGrainVolume>();
+
+/**
+ * Compute the average grain area in a polycrystal
+ */
+class AverageGrainVolume : public GeneralPostprocessor,
```

Comment:
MOOSE's lack of being able to resolve dependencies. This new object can couple to the GrainTracker which is a GeneralPostprocessor so it needs to run after that. Someday...